verkle-trie/src/database/generic.rs

```diff
@@ -108,7 +111,7 @@ impl<T: BareMetalKVDb> ReadOnlyHigherDb for GenericBatchDB<T> {
 
         self.inner
             .fetch(&labelled_key)
-            .map(|old_val_bytes| StemMeta::from_bytes(&old_val_bytes))
+            .map(|old_val_bytes| StemMeta::from_bytes(old_val_bytes.try_into().unwrap()).unwrap())
```

Comment:
Same question here for unwrap.

Reply:
Byte conversions are fallible, so we unfortunately must have unwraps for them, which I plan on slowly moving upwards through `Result`s